app/mailers/spool_submissions_report_mailer.rb

```diff
@@ -12,21 +12,17 @@ class SpoolSubmissionsReportMailer < ApplicationMailer
   ].freeze
 
   # Will add in Prod STEM recipients once 18815 approved in staging
```

Comment:
Do we still need this comment?

Reply:
I can go ahead and remove it, I do not see any similar comments within the file.